### PR TITLE
added web.xml + index.jsp + IndexController + dependency servlet-api …

### PR DESCRIPTION
**Pull request by Nick97-git**
…and plugin maven-war-plugin to pom.xml + Dao and Service layers using GenericDao and GenericService

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -24,6 +25,11 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>servlet-api</artifactId>
+            <version>2.5</version>
```
Comment (boroda4436) [inline pom.xml]: Let's use version 3.x

File: src/main/java/mate/academy/internet/shop/service/impl/OrderServiceImpl.java [hunk 2]
```diff
@@ -22,7 +22,7 @@
     public Order completeOrder(List<Product> products, User user) {
         List<Product> productList = new ArrayList<>(products);
         shoppingCartService.clear(shoppingCartService.getByUserId(user.getId()));
-        return orderDao.create(productList, user);
+        return create(new Order(productList, user));
```
Comment (boroda4436) [inline src/main/java/mate/academy/internet/shop/service/impl/OrderServiceImpl.java]: Let's complete order, then clear the shopping cart

File: src/main/webapp/WEB-INF/views/index.jsp [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: Lenovo
```
Comment (boroda4436) [inline src/main/webapp/WEB-INF/views/index.jsp]: @Nick97-git please remove the comments from JSP

**Review by boroda4436 — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 4]
```diff
@@ -24,6 +25,11 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>servlet-api</artifactId>
+            <version>3.0-alpha-1</version>
```
Comment (boroda4436) [inline pom.xml]: <dependency>
            <groupId>javax.servlet</groupId>
            <artifactId>javax.servlet-api</artifactId>
            <version>3.1.0</version>
        </dependency>
please, use this dependency

Comment (boroda4436) [reply pom.xml]: alpha releases can be unstable

Comment (Nick97-git) [reply pom.xml]: Understood

**Review by boroda4436 — APPROVED** [review]

**Pull request merged**